source/scripts/task-list.js

```diff
@@ -0,0 +1,249 @@
+class TaskListWidget extends HTMLElement {
+
+    /*
```

Comment:
Make sure the comment format follows the JSDoc notation. It should start with `/**` (two stars) and end with `*/`